### PR TITLE
add fp16 option when freeze graph

### PR DESCRIPTION
**Pull request by Cjkkkk**
add fp16 option when freeze graph.

### CODE REVIEW

File: models/tensorflow/nnf_tf_freezer/nnf_tf_freezer.py [hunk 1]
```diff
@@ -15,10 +15,11 @@
 from tensorflow.core.framework.tensor_shape_pb2 import TensorShapeProto
 from tensorflow.tools import graph_transforms
 from typing import List
+from convert_graph_fp16 import* 
 
 class nnf_tf_freezer(object):
     def __init__(self, frozen_graph= "frozen_graph.pb", const_folding=True, run_graph=True, xla=False, parallel=0,
-    warmup=5, num_iter=10, run_const_folded_graph=False, debug=False, is_training=False):
+    warmup=5, num_iter=10, run_const_folded_graph=False, debug=False, is_training=False, is_fp16=False):
```
Comment (wenxcs) [inline models/tensorflow/nnf_tf_freezer/nnf_tf_freezer.py:22]: What this param actually do is to covert graph to fp16graph, I think to_fp16 is more accurate.

**Review by wenxcs — COMMENTED** [review]
Did we use this sctip to generate a fp16 model and use TF to check if the result is right?

**Comment by Cjkkkk** [comment]
> Did we use this sctip to generate a fp16 model and use TF to check if the result is right?

I only ran alexnet, the result is : 
fp_32:
[ 0.00091444 -0.03034077 -0.00666213 -0.01023792  0.01407967  0.02431088
  0.00683159 -0.03537033  0.01791996  0.03885575] ...(size= 1001 end with 0.022596955 )

fp_16:
[ 0.000899 -0.03035  -0.006653 -0.010216  0.014084  0.0243    0.00684
 -0.0354    0.01793   0.03882 ] ...(size= 1001 end with 0.02258 )

I also view the graph in tensorboard, it seems that operations are converted to fp16 as expected.

File: models/tensorflow/nnf_tf_freezer/nnf_tf_freezer.py (hunk 1 above)
Comment (Cjkkkk) [reply models/tensorflow/nnf_tf_freezer/nnf_tf_freezer.py:22]: Sure, I will fix this.

**Review by wenxcs — APPROVED** [review]

**Pull request merged**